packages/fxa-content-server/tests/functional/pairing.js

```diff
@@ -63,6 +66,14 @@ function getQrData(buffer) {
   });
 }
 
+const generateCode = (secret) => {
+  secret = secret.replace(/[- ]*/g, '');
+  const authenticator = new otplib.authenticator.Authenticator();
+  authenticator.options = otplib.authenticator.options;
+  const code = authenticator.generate(secret);
+  return code;
+};
```

Comment:
I'm surprised we don't have a shared helper for this from testing the existing TOTP flows, worth extracting into one?